frontend/src/modules/tag/components/tag-popover.vue

```diff
@@ -1,36 +1,18 @@
 <template>
-  <app-dialog
-    v-if="computedVisible"
-    v-model="computedVisible"
-    title="Edit tags"
-    :pre-title="modelValue?.displayName ?? ''"
-  >
+  <app-dialog v-if="computedVisible" v-model="computedVisible" title="Edit tags">
```

Comment:
We should still render `pre-title` with the member's name when we are just editing tags for one member